ci/cloudbuild/builds/checkers.sh

```diff
@@ -191,38 +191,40 @@ time {
   done
 }
 
-printf "%-30s" "Updating generated doxygen:" >&2
-# Update the service's endpoint environment variables.
-for dox in google/cloud/*/doc/main.dox; do
-  lib="${dox%/doc/main.dox}"
-  inject_start="<!-- inject-endpoint-env-vars-start -->"
-  inject_end="<!-- inject-endpoint-env-vars-end -->"
-  env_vars=("")
-  for option_defaults_cc in "${lib}"/internal/*_option_defaults.cc; do
-    service="$(basename "${option_defaults_cc}" _option_defaults.cc)"
-    connection_h="${lib}/${service}_connection.h"
-    # Should we generate documentation for GOOGLE_CLOUD_CPP_.*_AUTHORITY too?
-    variable_re='GOOGLE_CLOUD_CPP_.*_ENDPOINT'
-    variable=$(grep -om1 "${variable_re}" "${option_defaults_cc}")
-    endpoint_re='"[^"]*?\.googleapis\.com"'
-    endpoint=$(grep -Pom1 "${endpoint_re}" "${option_defaults_cc}")
-    if grep -q 'location,' "${option_defaults_cc}"; then
-      endpoint="\"<location>-${endpoint:1:-1}\""
-    fi
-    make_connection_re='Make.*?Connection()'
-    make_connection=$(grep -Pom1 "${make_connection_re}" "${connection_h}")
-    env_vars+=("- \`${variable}=...\` overrides the")
-    env_vars+=("  \`EndpointOption\` (which defaults to ${endpoint})")
-    env_vars+=("  used by \`${make_connection}()\`.")
-    env_vars+=("")
-  done
-  sed -i -f - "${dox}" <<EOT
+printf "%-30s" "Updating doxygen landing pages:" >&2
```

Comment:
Thanks.

Totally optional: If you can think of something that begins with "Running", it would conform more with the existing output.

Reply:
Done. PTAL